.github/workflows/test-android-device.yaml

```diff
@@ -21,9 +21,13 @@ jobs:
       matrix:
         flutter-version: ['3.16.x']
         flutter-channel: ['stable']
-        device_model: ['oriole']
+        device_model: ['oriole', 'shiba']
```

Comment:
I think you do not need `device_model` and `os` in `matrix` now. Having them in `matrix.include` is enough.

Not sure though.

Reply:
I guess you're right. Afaik it will be included in the config anyway.